client/hosting/overview/components/migration-overview/components/migration-started-diy.tsx

```diff
@@ -0,0 +1,22 @@
+import { translate } from 'i18n-calypso';
+import Cards from './cards';
+import { Container, Header } from './layout';
+import type { SiteDetails } from '@automattic/data-stores';
+
+export const MigrationStartedDIY = ( { site }: { site: SiteDetails } ) => {
```

Comment:
We should call out this refactor on the PR description, as I was not expecting to see anything related to the Started status by reading the PR description.

Reply:
Answered it here https://github.com/Automattic/wp-calypso/pull/96160#issuecomment-2468373156